charts/metrics-agent/Chart.yaml

```diff
@@ -14,8 +14,8 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 2.11.36
+version: 2.11.37
```

Comment:
You missed one version in values.yaml. It is what makes the helm chart deployment pull in the correct image

Reply:
thanks i'll update that now